main.nf

```diff
@@ -166,7 +160,6 @@ summary['Max Time']     = params.max_time
 summary['IGDB Path']    = params.igblast_base
```

Comment:
You could print some more of the workflow options here,  right now you print very few.